### PR TITLE
Drop gratia-consumer (SOFTWARE-3218)

### PR DESCRIPTION
**Pull request by matyasselmeci**

### CODE REVIEW

File: osg_configure/configure_modules/rsv.py [hunk 1]
```diff
@@ -1086,70 +1082,6 @@ def split_2d_list(self, item_list):
         # We shouldn't reach here, but just in case...
         return array
 
-    def _set_gratia_collector(self, collector):
```
Comment (edquist) [inline osg_configure/configure_modules/rsv.py]: Isn't this section for the gratia (ie, gracc) collector?  (As opposed to the gratia-consumer in rsv?)

Comment (matyasselmeci) [reply osg_configure/configure_modules/rsv.py]: Nope. The configuration for gratia stuff lives in gratia.py.

Comment (edquist) [reply osg_configure/configure_modules/rsv.py]: Oh wow, so all this junk is just for the metric probe.

Ok, good riddance then!

**Comment by matyasselmeci** [comment]
There should be an OSG drinking game. Every time someone gets confused by the rsv gratia-consumer/gratia-collector thing, take a shot.

**Comment by edquist** [comment]
So, what is this PR, then? A 12-step program?

**Pull request merged**